apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -1312,3 +1352,15 @@ validator_string_re(Val, RE, Error) ->
     catch
         _:_ -> {error, Error}
     end.
+
+default_log_handlers() ->
+    #{
+        <<"file_handlers">> =>
+            #{
+                <<"default">> =>
+                    #{
+                        <<"level">> => <<"info">>,
```

Comment:
Here, `default` has level `info`, but [here](https://github.com/emqx/emqx/pull/10547/files#diff-49e4178ff28f46d34760aa4181bc135529d8bbdcffedd491fd4927f85b9ace1dR880) it is `warning`.

Perhaps this inner map containing `default` could be extracted to another function and used in both places to avoid divergence?

Reply:
this was stale code forgot to delete. it's already deleted in my latest push.